Dockerfile

```diff
@@ -119,6 +119,17 @@ COPY --chown=kat:kat katacomb/requirements.txt /tmp/requirements.txt
 ENV PATH="$PATH_PYTHON3" VIRTUAL_ENV="$VIRTUAL_ENV_PYTHON3"
 RUN install-requirements.py -d ~/docker-base/base-requirements.txt -r /tmp/requirements.txt
 
+# Install validation package
+ENV VALIDATION_REPO https://github.com/ska-sa/MeerKAT-continuum-validation.git
+ENV VALIDATION_BASE_PATH=/home/kat/valid
+
+# Retrieve validation package
+RUN mkdir -p $VALIDATION_BASE_PATH && \
+    git clone $VALIDATION_REPO ${VALIDATION_BASE_PATH}
+
+# Install validation package dependencies
+RUN pip install -r $VALIDATION_BASE_PATH/requirements.txt
+
```

Comment:
Better to put these requirements into the requirements.txt for katacomb and remove these lines since currently these continuum validation requirements will override and possibly break (if they are changed at some point in future) the requirements for katacomb.

Reply:
Done